gengogettext.py

```diff
@@ -87,26 +88,32 @@ def check_entry(lang, entry):
             return 'updated', None
         return 'waiting', None
 
+    job = get_job_data(entry.msgid, lang, edit_jobs)
+    if entry.msgstr:
+        job['comment'] += ('\nFuzzy translation. Previous translation was:\n' +
```

Comment:
What is 'fuzzy translation'?

Reply:
https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html

I've kept the existing logic with fuzzy entries